content/blog/2025/04/2025-04-08-morg-now-running-mas.md

```diff
@@ -0,0 +1,76 @@
++++
+date = "2025-04-08T15:30:00Z"
+title = "Matrix.org is now running MAS!"
+
+[taxonomies]
+author = ["Quentin"]
+category = ["matrix.org homeserver"]
++++
+
+We're thrilled to announce that the migration of matrix.org to the Matrix Authentication Service (MAS) is complete and went according to plan - having been running for over 24h in our brave new world, we’re declaring the migration a success! As of Monday April 7th 07:30 UTC, matrix.org is running on Matrix’s [next-generation auth system](https://github.com/matrix-org/matrix-spec-proposals/blob/hughns/delegated-oidc-architecture/proposals/3861-next-generation-auth.md) based on OAuth 2.0/OpenID Connect.
+
+This is no mean feat - the migration shifted all 45M access tokens and 110M users from Synapse to MAS in under 30 minutes (thanks in part to MAS’s cheeky use of the x86-64-v2 architecture; who knew that database migrations can be SIMD-accelerated?) - and represents the culmination of over 4 years of work to move Matrix to a modern authentication standard. Many thanks go to Element for funding, Hugh, Olivier and many other contributors who helped me make Next Gen Auth happen!
```

Comment:
Shouldnt Quentin mention himsef here? After all he also did a lot here :)

Reply:
I think this sentence points out that Quentin has been doing the bulk of the work

> who helped me make Next Gen Auth happen!

If you want to phrase it in a way that puts more emphasis on Quentin, please suggest :)

Reply:
ah missed the "me"